test/utils/server.js

```diff
@@ -14,13 +14,28 @@ var hostPort  = 'http://localhost:7231';
 var baseURL   = hostPort + '/en.wikipedia.test.local/v1';
 var bucketURL = baseURL + '/page';
 
+function loadConfig(path) {
+    var confString = fs.readFileSync(path).toString();
+    var backendImpl = process.env.RB_TEST_BACKEND;
+    if (backendImpl) {
+        if (backendImpl !== 'cassandra' && backendImpl !== 'sqlite') {
+            throw new Error('Invalid RB_TEST_BACKEND env variable value. Allowed values: "cassandra", "sqlite"');
+        }
+        if (backendImpl === 'sqlite') {
+            confString = confString.replace('restbase-mod-table-cassandra', 'restbase-mod-table-sqlite')
```

Comment:
@gwicke In the structure we ether need to 'know' the exact place where to make a replace (which's bad as we might change it and forget to change it here), or walk over all the structure (which looked slower to me). I'll test which way is faster